tests/transactional/check_phub.pm

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test packagehub repo on sle micro
+#
+# 1. Check if PackageHub extention is availble on sle micro
+# 2. Activate PackageHub extention
+# 3. Install a package from PackageHub repo
+#
+# Maintainer: qac <qa-c@suse.de>
+
+package check_phub;
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use version_utils 'is_sle_micro';
+use transactional;
+
+sub run {
+    select_console 'root-console';
+    my $out = script_output("transactional-update --quiet register --list-extensions");
+    if ($out =~ m#PackageHub/(.*)/#) {
+        trup_call("register -p PackageHub/$1/" . get_var('ARCH'));
+        process_reboot(trigger => 1);
+        trup_call("pkg install sshpass");
+        process_reboot(trigger => 1);
+        validate_script_output("rpm  -qa|grep sshpass", sub { m/sshpass/ });
+    }
+    else {
+        record_info('AVC', "PackageHub Unavaible", result => 'softfail');
+    }
+}
+
+sub post_fail_hook {
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
```

Comment:
Is this necessary? I am only questioning from my ignorance

Reply:
No. removed